staging/src/k8s.io/cli-runtime/pkg/printers/terminal.go

```diff
@@ -37,3 +41,35 @@ func WriteEscaped(writer io.Writer, output string) error {
 func EscapeTerminal(in string) string {
 	return terminalEscaper.Replace(in)
 }
+
+// IsTerminal returns whether the passed object is a terminal or not
+func IsTerminal(i interface{}) bool {
```

Comment:
I think we can do this movement in separate PR as a preparation of this PR.

Reply:
This commit already looks good to me. But if you don't have time, I can also take over it and open PR.